src/renderer/core/assets.json

```diff
@@ -0,0 +1,281 @@
+{
+  "assets": [
```

Comment:
This isn't a problem now but when our game is out of alpha or at least is being used by people we'll likely need to version our assets.json so if we make changes that would break previous assets packs it can be fixed.